### ISSUE TITLE
a poorly indented config file results in a incomprehensible message : "Unexpected Exception : 'NoneType' object has no attribute 'items'

### ISSUE THREAD

**Issue by PhilippeGregoire** [cea-hpc/milkcheck#61]
Here the cmd line is not well indented. This is not detected by yamllint command.

the command should report the error more clearly and in a way that help user to find it.
```
# cat bugindent.yaml 

services:
    hello:
        actions:
            start:
            cmd: echo Hello World

# yamllint bugindent.yaml 
bugindent.yaml
  1:1       error    too many blank lines (1 > 0)  (empty-lines)
  2:1       warning  missing document start "---"  (document-start)


#  milkcheck -c $PWD --dry-run
No actions specified, checking configuration...
[11:46:40] ERROR    - Unexpected Exception : 'NoneType' object has no attribute 'items'
[

#  milkcheck -c $PWD --dry-run --debug
[11:46:49] DEBUG    - Configuration
assumeyes: False
config_dir: /root/gregoirep/tmp/milkbug
confirm_actions: []
dryrun: True
fanout: 256
nodeps: False
report: full
reverse_actions: ['stop']
summary: False
tags: set([])
verbosity: 5
No actions specified, checking configuration...
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/MilkCheck/UI/Cli.py", line 560, in execute
    self.manager.load_config(self._conf['config_dir'])
  File "/usr/lib/python2.7/site-packages/MilkCheck/ServiceManager.py", line 192, in load_config
    self.fromdict(load_from_dir(conf))
  File "/usr/lib/python2.7/site-packages/MilkCheck/Engine/ServiceGroup.py", line 359, in fromdict
    service.fromdict(props)
  File "/usr/lib/python2.7/site-packages/MilkCheck/Engine/Service.py", line 253, in fromdict
    action.fromdict(props)
  File "/usr/lib/python2.7/site-packages/MilkCheck/Engine/Action.py", line 473, in fromdict
    BaseEntity.fromdict(self, actdict)
  File "/usr/lib/python2.7/site-packages/MilkCheck/Engine/BaseEntity.py", line 714, in fromdict
    for item, prop in entdict.items():
AttributeError: 'NoneType' object has no attribute 'items'

```
